Jetcaster/wear/src/main/java/com/example/jetcaster/ui/library/LibraryScreen.kt

```diff
@@ -62,166 +75,205 @@ fun LibraryScreen(
+            ) {
+                Text(stringResource(R.string.entity_no_featured_podcasts))
             }
-            if (topPodcasts.isNotEmpty()) {
-                items(topPodcasts.take(3)) { podcast ->
-                    PodcastContent(
-                        podcast = podcast,
-                        downloadItemArtworkPlaceholder = rememberVectorPainter(
-                            image = Icons.Default.MusicNote,
-                            tintColor = Color.Blue,
-                        ),
-                        onClick = {
-                            onTogglePodcastFollowed(podcast.uri)
-                        },
-                    )
-                }
-            } else {
-                item {
-                    PlaceholderChip(
-                        contentDescription = "",
-                        colors = ChipDefaults.secondaryChipColors(),
-                    )
-                }
+        }
+        if (topPodcasts.isNotEmpty()) {
+            items(topPodcasts.take(3)) { podcast ->
+                PodcastContent(
+                    podcast = podcast,
+                    podcastArtworkPlaceholder = painterResource(id = R.drawable.music),
+                    onClick = {
+                        onTogglePodcastFollowed(podcast.uri)
+                    },
+                )
+            }
+        } else {
+            item {
+                PlaceholderButton()
             }
         }
     }
 }
 
 @Composable
-private fun PodcastContent(
-    podcast: PodcastInfo,
-    downloadItemArtworkPlaceholder: Painter?,
-    onClick: () -> Unit,
-    modifier: Modifier = Modifier,
-) {
+private fun PodcastContent(podcast: PodcastInfo, onClick: () -> Unit, podcastArtworkPlaceholder: Painter?, modifier: Modifier = Modifier) {
     val mediaTitle = podcast.title
 
-    Chip(
-        label = mediaTitle,
-        onClick = onClick,
-        modifier = modifier,
-        icon = CoilPaintable(podcast.imageUrl, downloadItemArtworkPlaceholder),
-        largeIcon = true,
-        colors = ChipDefaults.secondaryChipColors(),
+    FilledTonalButton(
+        label = {
+            Text(
+                mediaTitle,
+                maxLines = 2,
+                overflow = TextOverflow.Ellipsis,
+            )
+        },
+        onClick = { onClick() },
+        icon = {
+            AsyncImage(
+                model = podcast.imageUrl,
+                contentDescription = stringResource(R.string.latest_episodes),
+                contentScale = ContentScale.Crop,
+                error = podcastArtworkPlaceholder,
+                placeholder = podcastArtworkPlaceholder,
+                modifier = Modifier
+                    .size(
+                        ButtonDefaults.LargeIconSize,
+                    )
+                    .clip(CircleShape),
+
+            )
+        },
+        modifier = modifier.fillMaxWidth(),
     )
 }
 
 @Composable
 fun LibraryScreen(
```

Comment:
when is this screen used versus the first LibraryScreen Composable?

Reply:
this is not part of these set of changes for Material 3, this was already here. Can we keep improvements for a follow up PR to close up on the Material 3 migration?